src/modules/navigation/components/Navigation/NavigationList/NavigationList.tsx

```diff
@@ -33,7 +34,12 @@ const NavigationList: FC<NavigationListProps> = ({ currentPath = '', items, onOp
 
 	const renderTrigger = (item: NavigationItem, iconName: IconLightNames) => {
 		return (
-			<div className={clsx(styles['c-navigation__link--wrapper'], 'u-cursor-pointer')}>
+			<div
+				className={clsx(styles['c-navigation__link--wrapper'], 'u-cursor-pointer')}
+				onKeyDown={(e) => onKey(e, keysSpacebar, () => e.preventDefault())}
```

Comment:
to select an item in the menu i can press enter, but the space key does not work.
Shouldn't pressing space also activate the button?

Reply:
Can't have both; preventing default scrolling behaviour also prevents default event bubble-up behaviour.